linetools/analysis/linelimits.py

```diff
@@ -25,8 +28,9 @@ class LineLimits(object):
     vlim : Quantity array
       velocity limits for the line in km/s
     """
+
     @classmethod
-    def from_specline(cls, aline, zlim):
+    def from_specline(cls, aline, z, zlim):
```

Comment:
But now we require the absline to always have limits defined and redshift defined. Couldn't we set the limits 'from_specline' without the need to input `z`? i.e. 'z' will be read directly from `aline.z`